pages/spec/[slug].js

```diff
@@ -118,17 +118,21 @@ export default function PostPage({ frontmatter, content, codeSnippets }) {
     return newId;
   }
 
+  const extractLibraryNameFromContent = () => {
+    return content.split("\n")[0].substring(2);
+  }
+
   return (
     <>
       <Head>
         <meta name="description" content={frontmatter.description} />
         <meta name="keywords" content={frontmatter.keywords} />
 
-        <title>{frontmatter.title}</title>
+        <title>{frontmatter.title || extractLibraryNameFromContent()}</title>
 
         {/* <!--FB--> */}
         <meta property="og:type" content="article" />
-        <meta property="og:title" content={`Ballerina - ${frontmatter.title}`} />
+        <meta property="og:title" content={`Ballerina - ${frontmatter.title  || extractLibraryNameFromContent()}`} />
```

Comment:
Can you add 
<meta name="twitter:title" content={`Ballerina - ${frontmatter.title  || extractLibraryNameFromContent()}`} />
for twitter as well